components/ws-daemon/pkg/content/initializer.go

```diff
@@ -276,7 +289,12 @@ func RunInitializerChild() (err error) {
 		return err
 	}
 
-	rs := &remoteContentStorage{RemoteContent: initmsg.RemoteContent}
```

Comment:
this has to be under a condition ??

Reply:
The "newDirectAccess()" function has a switch statement between minio, gcp, and s3. It will pick up the correct remoteContent depending on the setting in our values.yaml